packages/fiber/src/three-types.ts

```diff
@@ -40,7 +40,7 @@ export type Object3DNode<T, P> = Overwrite<
   {
     position?: Vector3
     up?: Vector3
-    scale?: Vector3
+    scale?: Vector3 | number
```

Comment:
This should already be covered by:

https://github.com/pmndrs/react-three-fiber/blob/b1b95352ae057d9d76b97bfda893b20d9c278a30/packages/fiber/src/three-types.ts#L15-L17

Reply:
Idd, weird I got a type error on a mesh when using a number, but can't reproduce anymore